Week3/homework/js-exercises/fortuneTeller.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+{
+  // Exercise 3: Be your own fortune teller
+
+  // Write a function named tellFortune.
+  // It takes 4 arguments: number of children (number),
+  // partner's name (string), geographic location (string), job title (string).
+  const tellFortune = (numberOfChildren, partnersName, geographicLocation, jobTitle) => {
+    // Randomly select values from the arrays.
+    numberOfChildren = numChildren[Math.floor(Math.random() * numChildren.length)];
+    partnersName = partnerNames[Math.floor(Math.random() * partnerNames.length)];
+    geographicLocation = locations[Math.floor(Math.random() * locations.length)];
+    jobTitle = jobs[Math.floor(Math.random() * jobs.length)];
+
```

Comment:
Ah. This is not right.
a) we should try to avoid re-assigning function parameters but should make use of them in a read-only view.
b) You need to pass variables `numChildren`, `parterNames` etc as function arguments when calling the function on line 31.